devicetypes/smartthings/smartsense-motion-sensor.src/smartsense-motion-sensor.groovy

```diff
@@ -307,8 +314,9 @@ def configure() {
 		configCmds += zigbee.batteryConfig()
 	}
 	configCmds += zigbee.temperatureConfig(30, 300)
+	configCmds += zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS)
```

Comment:
We need to add a second read of the battery here. That was added recently (via the call to refresh) to work around an issue in a particular version of hubCore.

Reply:
updated